### PR TITLE
docs(cockpit): add doc for retrieving logs using logcli

### PR DESCRIPTION
**Pull request by nerda-codes**

### CODE REVIEW

**Review by bene2k1 — APPROVED** [review]

File: observability/cockpit/api-cli/querying-logs-with-logcli.mdx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  - [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+  - A Scaleway account logged into the [console](https://console.scaleway.com)
+  - [Created a token](/observability/cockpit/how-to/create-token/) with the `query` permission for logs
+  - External and/or internal logs to query
+  - [Installed LogCLI](https://grafana.com/docs/loki/latest/query/logcli/#installation)
+
+
+## Exporting your environment variables
+
+Open a terminal and run the following command to export your environment variables. Make sure that you replace `<DATA_SOURCE_URL>` and `<COCKPIT_TOKEN>` with your values.
```
Comment (nerda-codes) [inline observability/cockpit/api-cli/querying-logs-with-logcli.mdx:32]:
```suggestion
Open a terminal and run the following command to export your environment variables. Make sure that you replace `<DATA_SOURCE_URL>` and `<COCKPIT_TOKEN_SECRET_KEY>` with your values.
```

File: observability/cockpit/api-cli/querying-logs-with-logcli.mdx [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    <Message type="note">
+     You can retrieve your data source URL in the [Data sources tab](https://console.scaleway.com/cockpit/dataSource) under the **API URL** section.
+     <Lightbox src="scaleway-data-source-url.webp" alt="" />
+    </Message>
+
+## Retrieving logs
+
+### Retrieving an Object Storage bucket's logs
+
+Open a terminal and paste the following command to retrieve logs from your desired data source. For the sake of this documentation, we are retrieving logs from a Scaleway Object Storage bucket. You can replace `object_storage_bucket` with any other Scaleway resource type provided that the resource has logs available.
```
Comment (nerda-codes) [inline observability/cockpit/api-cli/querying-logs-with-logcli.mdx:48]:
```suggestion
Open a terminal and paste the following command to retrieve logs from your desired data source. For the sake of this documentation, we are retrieving logs from a Scaleway Object Storage bucket. You can replace `object_storage_bucket` with any other Scaleway resource type, provided that the resource has logs available.
```

File: observability/cockpit/api-cli/querying-logs-with-logcli.mdx [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    ```
+An output similar to the following should display:
+
+```
+2024/05/20 17:04:33 <DATA_SOURCE_URL>_range?direction=BACKWARD&end=1716390273303145000&limit=30&query=%7Bresource_type%3D%22object_storage_bucket%22%7D&start=1716386673303145000
+```
+
+### Retrieving a Kubernetes Kapsule cluster's logs
+
+Open a terminal and paste the following command to retrieve logs from your Kapsul cluster.
```
Comment (nerda-codes) [inline observability/cockpit/api-cli/querying-logs-with-logcli.mdx:61]:
```suggestion
Open a terminal and paste the following command to retrieve logs from your Kapsule cluster.
```

File: observability/cockpit/api-cli/querying-logs-with-logcli.mdx [hunk 4]
```diff
@@ -0,0 +1,106 @@
+---
+meta:
+  title: Retrieving logs programmatically using LogCLI
+  description: This page provides information on how to retrieve logs stored in Cockpit programmatically using LogCLI
```
Comment (bene2k1) [inline observability/cockpit/api-cli/querying-logs-with-logcli.mdx:4]: optimize description

File: observability/cockpit/api-cli/querying-logs-with-logcli.mdx [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+Open a terminal and paste the following command to retrieve logs from your Kapsule cluster.
+
+    ```
+    logcli query '{resource_type="kubernetes_cluster"}'
+    ```
+
+An output similar to the following should display:
+
+    ```
```
Comment (bene2k1) [inline observability/cockpit/api-cli/querying-logs-with-logcli.mdx:69]: maybe cut the output to a few lines

**Review by bene2k1 — APPROVED** [review]

**Review by jcirinosclwy — APPROVED** [review]

**Review by ofranc — APPROVED** [review]

File: observability/cockpit/api-cli/querying-logs-with-logcli.mdx [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+    2024-05-22T17:32:43+02:00 {component="kapsule-autoscaler"} I0522        1 pre_filtering_processor.go:57] Node scw-k8s-sharp-robinson-default-e110ac268b5546d should not be processed by cluster autoscaler (no node group config)
+    2024-05-22T17:32:43+02:00 {component="kapsule-autoscaler"} I0522        1 static_autoscaler.go:492] Calculating unneeded nodes
+    2024-05-22T17:32:43+02:00 {component="kapsule-autoscaler"} I0522        1 static_autoscaler.go:445] No unschedulable pods
+    2024-05-22T17:32:43+02:00 {component="kapsule-autoscaler"} I0522        1 filter_out_schedulable.go:87] No schedulable pods
+    2024-05-22T17:32:43+02:00 {component="kapsule-autoscaler"} I0522        1 filter_out_schedulable.go:177] 0 pods marked as unschedulable can be scheduled.
+    2024-05-22T17:32:43+02:00 {component="kapsule-autoscaler"} I0522        1 filter_out_schedulable.go:176] 0 pods were kept as unschedulable based on caching
+    2024-05-22T17:32:43+02:00 {component="kapsule-autoscaler"} I0522        1 filter_out_schedulable.go:137] Filtered out 0 pods using hints
```
Comment (nerda-codes) [inline observability/cockpit/api-cli/querying-logs-with-logcli.mdx:101]:
```suggestion
```

File: observability/cockpit/api-cli/querying-logs-with-logcli.mdx (hunk 4 above)
Comment (nerda-codes) [inline observability/cockpit/api-cli/querying-logs-with-logcli.mdx:4]:
```suggestion
  description: Discover how to use LogCLI to programmatically retrieve logs stored in Cockpit.
```

**Pull request merged**